src/Beman/InplaceVector/tests/CMakeLists.txt

```
@@ -5,7 +5,7 @@
 
 # Tests
 add_executable(
-        InplaceVectorTest.test)
+        InplaceVectorTest.test InplaceVector.test.cpp)
```

```suggestion
        Beman.InplaceVectorTest.test InplaceVector.test.cpp)
```
All targets need a Beman prefix in case this is incorporated in a larger CMake project. (targets are in a global namespace).